src/net.cpp

```diff
@@ -1571,8 +1579,53 @@ void StopMapPort()
 #endif
 
 
+void CConnman::DNSCachesUpdate() const
+{
+    // Only reachable nodes do this, because those are already easy to find.
+    // Unless the node is explicitly set to be non-reachable (-listen=0),
+    // the only way to check if it is reachable is to observe inbound connections.
+    // This check is made every time, because a node may become non-reachable
+    // due to external factors (e.g. local network settings update).
+    if (CountInboundConnections() == 0) return;
+
+    // Every time a new widely-requests service bit is deployed, this list should be
+    // updated to invalide records for that service bit.
+    std::vector<uint64_t> SERVICE_BITS_COMBINATIONS{
+        // Legacy nodes
+        NODE_NETWORK,
```

Comment:
Wouldn't these be requesting the bare name without any service bit specification?

Reply:
I think this should be x1.dns.whatever.com?